sw/host/cryptotest/testvectors/parsers/cryptotest_util.py

```diff
@@ -87,7 +87,7 @@ def parse_rsp(file_path: str, persists: list[str] = []) -> dict:
                 if key in group_variables:
                     # We've already seen this variable in the current group. Store all
                     # the values in a list.
-                    if type(d[key]) == str:
+                    if type(d[key]) is str:
```

Comment:
Generally isinstance is more idiomatic. Perhaps less important here, but it allows sub classes to be used as well as the parent classes. Again probably not an issue in this case as str is a built-in type.

Reply:
I think this is clearer in the case of `str` and it’s what Ruff suggested, so I’ll leave these case. Thanks for the tip though, I’ll use this for class types.